tests/scm/CMakeLists.txt

```diff
@@ -30,4 +31,9 @@ ADD_GUILE_TEST(SCMOpenogTestRunnerFail scm-opencog-test-runner-fail.scm)
 # The SCMOpenogTestRunnerFail fails purposely, as it is testing that the
 # scheme opencog-test-runner exits properly; thus requiring the flipping
 # of the test result by setting its properties.
-SET_TESTS_PROPERTIES(SCMOpenogTestRunnerFail PROPERTIES WILL_FAIL TRUE)
+SET_PROPERTY(TEST SCMOpenogTestRunnerFail PROPERTY WILL_FAIL TRUE)
```

Comment:
Well, if `SCMOpenogTestRunnerPass` below needs `PROPERTY ENVIRONMENT "GUILE_LOAD_PATH=${PROJECT_SOURCE_DIR}/opencog/scm")` then so does `SCMOpenogTestRunnerFail` -- both should look the same, from this point of view

Reply:
Yes, I have fixed it in later commit.

Reply:
oh duh kant read